org.eclipse.dawnsci.nexus.test/src/org/eclipse/dawnsci/nexus/NexusTestServiceHolder.java

```diff
@@ -20,15 +20,15 @@
 	
 	private static INexusFileFactory nexusFileFactory;
 
-	public static void setNexusBuilderFactory(NexusBuilderFactory nexusBuilderFactory) {
+	public void setNexusBuilderFactory(NexusBuilderFactory nexusBuilderFactory) {
 		NexusTestServiceHolder.nexusBuilderFactory = nexusBuilderFactory; 
 	}
 	
 	public static NexusBuilderFactory getNexusBuilderFactory() {
 		return nexusBuilderFactory;
 	}
 	
-	public static void setNexusFileFactory(INexusFileFactory nexusFileFactory) {
+	public void setNexusFileFactory(INexusFileFactory nexusFileFactory) {
 		NexusTestServiceHolder.nexusFileFactory = nexusFileFactory; 
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)